### ISSUE TITLE
Incorrect IM versions specified on some pages

### ISSUE THREAD

**Issue by lkintact** [ImageMagick/Website#112]
As far as I understand, the latest IM version is **7.1.1-39**, so the versions on these pages are specified incorrectly:

* On https://usage.imagemagick.org/masking/:
1. The "-alpha Remove" method (added to IMv**7.7.5**) is designed to remove the transparency from an image, using the current "-background". 

* On https://usage.imagemagick.org/warping/:
1. As of IMv**7.7.3-4** the Rotate Operator is now using Distort Operator and Scale-Rotate-Translate (SRT) Distortion.
Here is the more direct rotation using the underlying SRT Distortion.
2. Before IMv**7.7.3-4** the Rotate Operator was implemented using Simple 90 rotation Distorts followed by three Image Shears, a technique known as 'Rotate by Shear' (RBS).
3. Update: As of IMv**7.7.3-4** the Rotate Operator is now using Distort Operator and SRT Distortion. As such the previous example will now produce the much less blury result of the next example.

* On https://usage.imagemagick.org/misc/:
1.  Update: As of IMv**7.7.3-4** the rotate operator is now internally using Distort Operator, so the above many no longer be true.

**Comment by fmw42** [comment]
Those IM versions are correct. They indicate when the new feature was added so that users know. They should not be updated to the current version.

**Comment by lkintact** [comment]
> They should not be updated to the current version.

I didn't mean they should, I meant that those versions are non-existent...

**Comment by fmw42** [comment]
Yes, you are correct. 7.7.x.x is not correct. IM is currently only at 7.1.x.x
